### DISCUSSION TITLE
UI Bug

### DISCUSSION THREAD

**Discussion by 45i** [community/community#4691] [Other Feature Feedback, Questions, & Ideas]
How to reproduce?
- Open up a repo
- Create or open a project in that repo (Projects tab)
- Click around a bunch of times
- Come back to the Code tab
Notice how the padding has changed

Heres a screenshot: 
![image](https://user-images.githubusercontent.com/86361659/125960458-94e172f6-010d-4b7e-8639-22badbbe3ee1.png)

**Comment by MathiasReker** [comment]
Hello. You can report the bug here: https://support.github.com/contact/bug-report
